app/commands/command_localised_time.js

```diff
@@ -0,0 +1,16 @@
+import { 
+    getFormattedDateTimeString,
+    getUnixTimeInSeconds,
+} from "../utils/time_tools.js";
+
+export function getDiscordFormattedUnixTime(options) {
```

Comment:
(suggestion: architecture) I could see this useful to have this as a general function, as for example the sweats messages would also need to convert the time into discord format.
Maybe you could put this into the `time_tools.js` File too, and just call it here?